### PR TITLE
Release v1.6.0

### PR DESCRIPTION
**Pull request by khalifaa55**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -256,9 +262,9 @@ The following roadmap covers the main features and ideas we want to implement bu
 
 - [x] Support Erigon on Gnosis
 - [x] Support for Lido CSM
+- [x] Support for Nimbus client as Consensus and Validator
 - [ ] Include monitoring tool for alerting, tracking validator balance, and tracking sync progress and status of nodes
```
Comment (stdevMac) [inline README.md:266]: We are delivering this in this release :)

File: cmd/lido-exporter/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Lido Exporter is a service that exports data from the Lido CSM smart contracts as Prometheus metrics.
+
+## Features
+
+- Collects metrics for penalties, exit requests, node operator info, keys info, bond info, and rewards info
+- Supports Holesky network (Mainnet support can be easily added)
+- Exports metrics in Prometheus format
+
+## Usage
+
```
Comment (stdevMac) [inline cmd/lido-exporter/README.md:12]: We should add here how to run it using the docker image published on DockerHub

**Review by stdevMac — CHANGES_REQUESTED** [review]

File: cmd/lido-exporter/README.md (hunk 2 above)
Comment (khalifaa55) [reply cmd/lido-exporter/README.md:12]: [Done](https://github.com/NethermindEth/sedge/pull/456/commits/466d65f5b91ba8357e450b3025905668920e7285)

File: README.md (hunk 1 above)
Comment (khalifaa55) [reply README.md]: [Done](https://github.com/NethermindEth/sedge/pull/456/commits/466d65f5b91ba8357e450b3025905668920e7285)

**Review by stdevMac — APPROVED** [review]

**Pull request merged**